weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/callsTableQuery.ts

```diff
@@ -120,20 +125,87 @@ export const useCallsForQuery = (
     callsStats.refetch();
   }, [calls, callsStats, costs]);
 
+  // query for annotation feeback
+  const {useFeedbackQuery} = useWFHooks();
+  const feedbackQueryQuery = useMemo(() => {
```

Comment:
why not just do `include_feedback` and filter the results in the client?

Reply:
I tried both, I thought this way would be more performant: we can get a faster first-paint, then fill in feedback

Reply:
i see. makes sense. I think the feedback type filter might be better to do client side since:
1. The data size for feedback is quite small per row (so bandwidth is not a concern as much)
2. You will want to do more than just annotation really soon

Reply:
yep that makes sense ez pivot